### PR TITLE
Fixed sai thrift set behavior.

### PR DESCRIPTION
**Pull request by anton7811**
- Fixed sai thrift set() method argument name
- Added SAI status converter from code to string
- Added SaiStatus Enum

Signed-off-by: Anton Putria <anton.putrya@plvision.eu>

### CODE REVIEW

**Comment by akokhan** [comment]
Please provide PR description

File: common/sai_data.py [hunk 1]
```diff
@@ -1,126 +1,18 @@
 import json
 from enum import Enum
+from sai_thrift.sai_headers import *
```
Comment (andriy-kokhan) [inline common/sai_data.py:3]: I think this will break Redis case.

Comment (anton7811) [reply common/sai_data.py:3]: Good point!
I'll think about how to do it differently. I'll appreciate it if you have any suggestions.

Comment (andriy-kokhan) [reply common/sai_data.py:3]: As we discussed earlier, SAI metadata has to be used instead
https://github.com/opencomputeproject/SAI-Challenger/blob/2d58df2fe75eab5246342bf1689c887a95b22a3f/common/sai.py#L179
One approach for both Redis and Thrift flows

Comment (andriy-kokhan) [reply common/sai_data.py:3]: Well... actually not sure whether the list of object types is in metadata. We should check. That should be the right approach.

Comment (anton7811) [reply common/sai_data.py:3]: As I understand there is already a task in the backlog to fix this for sai_thrift_metadata.py.
Maybe to keep things clear the best way to resolve this is to:
1. Keep SaiObjType and SaiStatus as constant Enums for this PR
2. Switch to autogenerating in a separate PR for both sai_data.py and sai_thrift_metadata.py

Comment (andriy-kokhan) [reply common/sai_data.py:3]: Both options are fine to me.

Comment (anton7811) [reply common/sai_data.py]: Updated.

**Review by andriy-kokhan — APPROVED** [review]

**Pull request merged**